unit-tests/src/test/java/org/eclipse/collections/impl/tuple/TuplesTest.java

```diff
@@ -61,6 +81,19 @@ public void equalsHashCode()
         Assert.assertNotEquals(pair1, new Object());
     }
 
+    @Test
+    public void equalsHashCodeTriple()
+    {
+        Triplet<String> triple1 = Tuples.triplet("1", "1", "1");
+        Triple<String, String, String> triple1a = Tuples.triple("1", "1", "1");
+        Triple<String, String, String> triple2 = Tuples.triple("2", "2", "2");
```

Comment:
```suggestion
        Triplet<String> triplet = Tuples.triplet("1", "1", "1");
        Triple<String, String, String> triple1 = Tuples.triple("1", "1", "1");
        Triple<String, String, String> triple2 = Tuples.triple("2", "2", "2");
```

Reply:
I made the test similar to the test for tuples (pair/twin) so it is consistent